src/common/libraries/PricingLib.sol

```diff
@@ -251,6 +251,17 @@ library PricingLib {
         return convertWithReciprocalPrice(poolAmount, poolDecimals, assetDecimals, pricePoolPerAsset, rounding);
     }
 
+    /// @dev Converts asset amount to pool amount.
```

Comment:
If this function is only used in tests, I don't think it should be in the src contract. It should be a test helper then. Otherwise it is exposed to audits etc.

Reply:
That is a fair point!